pkg/systemd/logs.html

```diff
@@ -52,10 +52,14 @@
         </button>
         <ul id="prio-lists" class="dropdown-menu" role="menu">
           <li><a data-prio="*" translatable="yes">Everything</a></li>
-          <li><a data-prio="5" translatable="yes">Problems, Errors, Warnings, Notices</a></li>
-          <li><a data-prio="4" translatable="yes">Problems, Errors, Warnings</a></li>
-          <li><a data-prio="3" translatable="yes">Problems, Errors</a></li>
-          <li><a data-prio="2" translatable="yes">Only Problems</a></li>
+          <li><a data-prio="0" translatable="yes">Emergency</a></li>
+          <li><a data-prio="1" translatable="yes">Alert</a></li>
```

Comment:
It's a bit unclear if this will only show Alert, or Alerts and Emergency, so I would suggest this to be "Alert and above".
Same for Critical, Error, Warning, Notice, Info and Debug.